### DISCUSSION TITLE
How to convert a TIF with alpha channels in cmyk mode into a semi transparent PNG

### DISCUSSION THREAD

**Discussion by zxc155226522** [ImageMagick/ImageMagick#6227] [Help]
### ImageMagick version

ImageMagick-7.1.0-portable-Q8-x64

### Operating system, version and so on

win11

### Description

How to convert TIF with alpha channels in cmyk mode to semi transparent PNG? The following command can convert without storing transparent channels in PS, but if transparent channel conversion is stored, there will be problems
`magick "1 (1).tif[0]" -profile sRGB.icc -channel A -negate +channel out2.png`

https://drive.google.com/file/d/10oNHW2-Kf9PxAcDgwuEjRbUUU93PYmUi/view?usp=sharing
https://drive.google.com/file/d/12DcgLtTO8JH9kK6XbhNeugAWNyYAxaAE/view?usp=sharing
![image](https://user-images.githubusercontent.com/129932013/230003301-eb02fadf-14f7-40fe-82df-26855f9d79de.png)

**Comment by snibgo** [comment]
> How to convert TIF with alpha channels in cmyk mode to semi transparent PNG?

Your TIFF file has can embedded profile. For the first image in that file, with CMYKA channels, this command will convert it to sRGB:
```
magick 1.tif -profile sRGB.icc out.png
```
That is all you need to do. Your command also negates the alpha channel. Why do you do that?

**Reply by zxc155226522** [reply]
These two files are stored in different ways, the same command will have problems, so I want to see if there is a way to distinguish whether there is storage transparency
